app/models/concerns/authorizers/image.rb

```diff
@@ -9,5 +9,9 @@ def can_destroy_image?(image)
       return true if can_admin_system?
       image.record == user
     end
+
+    def can_create_sponsor_svg?
```

Comment:
We may want to allow this. We just need to have the creative status as "active". As long as it remains pending and locked when active, I think we can open this up.